pkgs/applications/graphics/imv/default.nix

```diff
@@ -1,46 +1,37 @@
-{ stdenv, fetchFromGitHub
-, freeimage, fontconfig, pkgconfig
-, asciidoc, docbook_xsl, libxslt, cmocka
-, librsvg, pango, libxkbcommon, wayland
-, libGLU, icu
-}:
+{ stdenv, fetchFromGitHub, freeimage, fontconfig, pkgconfig, asciidoc
```

Comment:
Did you run this file into a formatter? If there is no particular reason I would keep fewer packages per line, to make the diff easier to read.

Reply:
yes it's from `nixfmt`, I don't have a strong opinion though.  On the one hand in the future it might be better if `nixfmt` is used again, on the other side the diff is now a bit harder to read :shrug:

Reply:
Uhm, the tools is pretty new and has still a few problems: there is at least another issue with function arguments, https://github.com/serokell/nixfmt/issues/46. This formatting, in particular, is in contrast with the Nixpkgs [guide](https://nixos.org/manual/nixpkgs/stable/#chap-conventions) and the general trend of keeping short lines, for example https://github.com/NixOS/nixpkgs/commit/786d9316ad2dd77ebc9e72e47f64698095b25962 https://github.com/NixOS/nixpkgs/commit/066412597a0c49ce0a77f47a9352420a09c80d59.

A more appropriate tool for Nixpkgs may be [nixpkgs-fmt](https://github.com/nix-community/nixpkgs-fmt).

Reply:
Thanks for the pointer to `nixpkgs-fmt`, I ran that instead and pushed the changes